packages/commonwealth/server.ts

```diff
@@ -29,6 +31,11 @@ stats({
 blobStorage({
   adapter: S3BlobStorage(),
 });
+blobStorage({
```

Comment:
I keep thinking that the dual port option was cleaner (just adding a new contentStorage() port), but the multiple adapter by key solution also looks fine here 👍